Program.cs

```diff
@@ -1,10 +1,15 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿Console.WriteLine (armstrong ());
+
+string armstrong () {
+   Console.Write ("Enter a number to check whether it is an armstrong number or not: ");
+   if (int.TryParse (Console.ReadLine (), out int input)) {

```

Comment:
It is better to write code like `if (!int.TryParse ("", out int i)) return "Invalid input";` instead of embedding a whole block of code within. This will make it more readable.